app/views/landing-pages/parents/PageParents.vue

```diff
@@ -1261,3 +1261,9 @@ export default {
 }
 
 </style>
+
+<style>
```

Comment:
Two small changes.

1. Could we put this in the scoped style tag block ( `<style scoped>`) above?
Unless we need to for something external to the component (i.e. tooltips), this leaves us at risk of the style leaking across the single page app.

2. Nitpick: isn't the `id h1` specificity enough to override other h1 rules? Is there a reason `!important` is needed?

Reply:
Added css inside `Scoped` style and put same css inside `PageParentsJumbotron.vue` component to make h1 bolder in `/parents` page.